test/unit/lib/plugins/aws/package/compile/events/httpApi.test.js

```diff
@@ -134,6 +134,7 @@ describe('lib/plugins/aws/package/compile/events/httpApi.test.js', () => {
               payload: '1.0',
               cors: true,
               metrics: true,
+              disableDefaultEndpoint: false,
```

Comment:
Let's remove this, and confirm in test that `DisableExecuteApiEndpoint` is not set